src/Core/src/AwsClientFactory.php

```diff
@@ -576,4 +577,17 @@ public function cognitoIdentityProvider(): CognitoIdentityProviderClient
 
         return $this->serviceCache[__METHOD__];
     }
+
+    public function athena(): AthenaClient
```

Comment:
could you add an entry in changelog :pray:

Reply:
I add an entry in the core changelog file, I am note sure if you where referencing that

Reply:
each folder is distributed in AA different package.
We need to add an entry in the Core CHANGELOG for this new method.

and also on entry in the SymfonyBundle for the new service